tests/test_datacard.py

```diff
@@ -0,0 +1,38 @@
+import audbcards
+import pytest
+import difflib
+import numpy as np
+
+from audbcards.core.dataset import create_datasets_page
+
+
+def test_datacard_from_template_lines_similar(db, default_template):
+    """Create datacard using jinja2 via Dataset and Datacard."""
+
+    dataset = audbcards.Dataset(pytest.NAME, pytest.VERSION)
+    dc = audbcards.Datacard(dataset)
+    content = dc._render_template()
+
+    ratios = []
+    for line1, line2 in zip(default_template.split("\n"), content.split("\n")):
+        sm = difflib.SequenceMatcher(lambda x: x == " ", line1, line2)
+        ratios.append(sm.ratio())
```

Comment:
Could you add a comment explaining roughly what this does?

I guess the expected RST file and the generated RST are not exactly identical so you need some special code for comparison? If yes, why are they not exactly identical?

Reply:
Ah, ok, found your comment below:

> `# there are lines that will differ (author, date of dataset)`

So maybe move this slightly up?

Reply:
I added a high level description of the assertions made to the docstring.

Reply:
Thanks.